src/coreComponents/linearAlgebra/unitTests/testLinearAlgebraUtils.hpp

```diff
@@ -426,6 +421,10 @@ void compute2DElasticityOperator( MPI_Comm const comm,
     globalIndex dofIndex[8];
     computeQuadElementDofIndices( iCell + iStart, nCellsX, dofIndex );
 
+    // Construct local stiffness matrix (same for all cells)
```

Comment:
```suggestion
    // Construct local stiffness matrix (same for all cells).
    // This could be done out of the `forAll` loop,
    // but a strange behavior of `gcc 9` in `Release` doesn't copy `Ke` properly.
```